packages/editor/src/components/post-taxonomies/flat-term-selector.js

```diff
@@ -288,8 +292,15 @@ export function FlatTermSelector( { slug, __nextHasNoMarginBottom } ) {
 		singularName
 	);
 
+	const Wrapper = ( { children } ) =>
+		__nextHasNoMarginBottom ? (
+			<VStack spacing={ 3 }>{ children }</VStack>
```

Comment:
```suggestion
			<VStack spacing={ 4 }>{ children }</VStack>
```

I _think_ we can just use the new standard spacing? No strong opinion though.

Reply:
I have no strong opinion on maintaining the original spacing, let's apply the new standard. 👍

![image](https://github.com/user-attachments/assets/d936243e-d1f0-441e-987a-a5415ae23129)